TEMPLATE.md

```diff
@@ -0,0 +1,81 @@
+---
+id:
+title:
+status: draft
+created:
+contributors:
+    -
+---
+
+## Preamble
+
+    Idea: <to be assigned>
+    Title: <title>
+    Status: <Draft | In Progress | Closed >
+    Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+    Requires (*optional): <Idea number(s)>
+    Replaces (*optional): <Idea number(s)>
+
+
+## Summary
+<!-- "If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the Idea. -->
+
+## Swarm Participants
+<!-- Each contributor pledges to the idea with their FOCUS value. (hours per week) -->
```

Comment:
Is all this prelude necessary? Might be easier to just have it be something like:

```
<!-- Fill in PM, UX(R), QA etc as needed/required -->

Lead Contributor:
Evaluator:
Contributor:
Contributor:
```

Reply:
If we want to get to more of a template, then the closer it is to a form, the easier for the person completing it. We could even identify mandatory items with * so they know the idea cant progress without it.

Reply:
+1 to Arash's suggestion

Reply:
I like that idea too. What do you think this would look like in this specific section, given that not all swarms are user-facing, but all ideas share the same template?

Reply:
removed pre-fills here, if someone has thoughts on how concretely it can look different (while still working for different swarm types) I'm all ears